src/pudl/package_data/notebooks/how_to_use_pudl_out.ipynb

```diff
@@ -0,0 +1,572 @@
+{
```

Comment:
<p>Missing full comment on the Rolling Average line.</p><p> </p><p>Need to point them to the EIA for an API key, since the monthly fill will fail mysteriously if they don't have one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/844/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>thanks! i had forgotten about this.</p>